app/forms/questionnaire_form.py

```diff
@@ -66,10 +85,40 @@ def _get_target_total_and_currency(self, calculation, question):
         target_answer = self.schema.get_answer(calculation['answer_id'])
         return self.answer_store.filter(answer_ids=[target_answer['id']]).values()[0], target_answer.get('currency')
 
-    def _validate_date_range_question(self, question_id, period_from_id, period_to_id):
+    def validate_date_range_with_period_limits_and_single_date_limits(self, question_id, period_limits, date_from, date_to):  # pylint: disable=too-many-locals
```

Comment:
Remove `# pylint: disable=too-many-locals`. It will pass without this. If it didn't pass Lint you should really address the code rather than add an ignore.